pkg/arvo/app/chat-hook.hoon

```diff
@@ -109,7 +103,8 @@
       =/  old-mailbox=mailbox
         (need (scry:cc (unit mailbox) %chat-store [%mailbox chat]))
       =*  enves  envelopes.old-mailbox
-      :~  (chat-poke:cc [%delete chat])
+      :~  (chat-poke:cc [%delete new-chat])
```

Comment:
Why are we concerned about these things already existing? Does anything create with `/~` prefix rn? (Commit says "fix ota chat backlog duplication" but I don't understand how we'd end up with duplication in the first place.)

If they exist, they'll pop up in the `turn  ~(tap in keys)` routine, meaning we'll create `/~/~` chats too. _If_ we want to deal with them, we should probably filter them out there and send deletes only?

Reply:
Depending on the order that the OTA occurs in and the order that the hook and store come online, the events may be sent twice, so we needed to make sure they were idempotent. They’ve been tested and we’re sure they’re idempotent.